src/util/FactorUtil.js

```diff
@@ -285,6 +285,27 @@ function (Okta) {
     return result.join(' ');
   };
 
+  fn.getCardinalityText = function (enrolled, required, policy) {
+    if (policy && policy.enrollment) {
+      var enrollmentInfo = policy.enrollment;
+      if (enrolled) {
+        return (enrollmentInfo.enrolled === 1) ? false :
+          Okta.loc('enroll.choices.cardinality.setup', 'login', [enrollmentInfo.enrolled]);
+      }
+      else if (required) {
+        return (enrollmentInfo.enrolled === 0) ?
+          Okta.loc('enroll.choices.cardinality.required', 'login', [enrollmentInfo.minimum]) :
+          Okta.loc('enroll.choices.cardinality.setup.remaining', 'login',
+            [enrollmentInfo.enrolled, enrollmentInfo.minimum]);
+      }
+      else {
+        return false;
+      }
+    }
+    else {
```

Comment:
nit: just `return false` at the end of the function and you can avoid a couple of `else { return false }` blocks